### PR TITLE
fix: CLIN-207 graphql with rpt token

### PR DESCRIPTION
**Pull request by creativeyann17**
- [x] fhir graphql with rpt token (instead of user token)
- [x] add token when request is built

### CODE REVIEW

File: client/src/store/providers/apollo.tsx [hunk 1]
```diff
@@ -21,19 +21,24 @@ const arrangerLink = createHttpLink({
...
 });
 
-const getAuthLink = (userToken: string) => setContext((_, { headers }) => ({
-  headers: {
-    ...headers,
-    authorization: userToken ? `Bearer ${userToken}` : '',
-  },
-}));
+const getAuthLink = () => (
+  setContext((_, { headers }) => (
```
Comment (creativeyann17) [inline client/src/store/providers/apollo.tsx:25]: TBC I'm not very familiar with ApolloLink, for what I understand 'setContext' will be called every times a request is done ?

Comment (francisl) [reply client/src/store/providers/apollo.tsx:25]: ca va être appelé a chaque re-render du apolloClient (don't a chaque update de données)

regarde la section de caching: https://www.apollographql.com/docs/react/api/link/apollo-link-context/#caching-lookups

**Review by francisl — CHANGES_REQUESTED** [review]
stp, valide pour le caching

**Review by francisl — APPROVED** [review]
:shipit:

**Pull request merged**